testdata/project-v4-with-deploy-image/config/prometheus/kustomization.yaml

```diff
@@ -1,2 +1,4 @@
 resources:
```

Comment:
Could we also update this one by adding info on the top? something like:

# These kustomization manifest built the required manifests to allow scrapping the metrics for
# the Prometheus when/if we are using Prometheus Operator. Note that inside of this config
# we can find the ServiceMonitor which configures the metrics endpoint and the rules required 
# to allow the Prometheus Operator have access to the service monitor

Reply:
added